<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLine</class>
 <widget class="QWidget" name="CLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Énoncé</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="le_solution"/>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QLineEdit" name="le_statement">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="le_crc">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>CRC</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Solution</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>le_solution</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FLine</receiver>
   <slot>verify(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_crc</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FLine</receiver>
   <slot>verify(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>verify(QString)</slot>
 </slots>
</ui>
